include/libopencm3/stm32/common/flash_common_f234.h

```diff
@@ -60,6 +60,12 @@
 #define FLASH_ACR_LATENCY_6WS		0x06
 #define FLASH_ACR_LATENCY_7WS		0x07
 
+#define FLASH_ACR_DCRST			(1 << 12)	/* Data cache reset */
+#define FLASH_ACR_ICRST			(1 << 11)	/* Instruction cache reset */
+#define FLASH_ACR_DCEN			(1 << 10)	/* Data Cache enable */
+#define FLASH_ACR_ICEN			(1 << 9)	/* Instruction Cache enable */
+#define FLASH_ACR_PRFTEN		(1 << 8)	/* Prefetch enable */
```

Comment:
a) these should be added to stm32/f4/flash.h if they are f4 specifc, and more importantly...
b) these all exist in https://github.com/libopencm3/libopencm3/blob/master/include/libopencm3/stm32/common/flash_common_f24.h#L52-L56 already :)  Are you sure you have an updated library locally? you should be getting errors about redefining things.

Reply:
Hmm, if you look at the master branch of the repo where I generate this PR from it shows it as up to date with the main repo. I did not pull it against a staging repo if that was the question.

Reply:
So just rebased and remerged this with the latest and I can see the duplicates.